lib/src/attributes/variants/variants.utils.dart

```diff
@@ -132,4 +130,28 @@ class VariantUtils {
       },
     );
   }
+
+  /// Short Utils: active
+  static Variant<T> active<T extends Attribute>() {
+    return Variant<T>(SystemVariants.active.value);
+  }
+
+  /// Short Utils: inactive
+  static Variant<T> inactive<T extends Attribute>() {
+    return Variant<T>(SystemVariants.inactive.value);
+  }
+
+  /// Short Utils: title
+  static Variant<T> title<T extends Attribute>() {
+    return Variant<T>(SystemVariants.title.value);
+  }
+
+  /// Short Utils: paragraph
+  static Variant<T> paragraph<T extends Attribute>() {
```

Comment:
Don't think these variants should be  here. Maybe they should get defined within the headless component.